includes/class-wcpdf-main.php

```diff
@@ -641,11 +641,13 @@ public function copy_directory ( $old_path, $new_path ) {
 
 		global $wp_filesystem;
 
-		require_once( ABSPATH . 'wp-admin/includes/file.php' ); // required for WP_Filesystem()
+		if ( ! function_exists( 'WP_Filesystem' ) ) {
+			require_once( ABSPATH . 'wp-admin/includes/file.php' );
+		}
 
-		// check for WP_Filesystem(), is required for copy_dir()
-		if ( ! WP_Filesystem() ) {
-			WP_Filesystem();
+		if ( ! WP_Filesystem() || is_null( WP_Filesystem() ) ) {
```

Comment:
`is_null`  is considered "falsy" so the second check would never be executed here (is redundant)